presto-main/src/test/java/com/facebook/presto/cost/TestFilterStatsCalculator.java

```diff
@@ -279,10 +279,12 @@ public void testNotStats()
                 .outputRowsCount(625) // FIXME - nulls shouldn't be restored
                 .symbolStats(new Symbol("x"), symbolAssert ->
                         symbolAssert.averageRowSize(4.0)
-                                .lowValue(0.0)
+                                .lowValue(-10.0)
```

Comment:
We should be able to realize `x>=0`. Add `// TODO should be 0.0`

Reply:
`<=` fixed the test